CONTRIBUTING.md

```diff
@@ -120,3 +125,11 @@ While not all code submitted at this time conforms to this standard, it is the i
     - `1.f`
     - `1.L`
   - No: `1` for `f32` or `f64`
+
+## <a name="prs"></a>Pull Requests
+
+If you just matched one function and don't think that you'll be matching anymore, you don't need to create a PR! You can just create an issue against the repo with your decomp.me link. See https://github.com/doldecomp/melee/issues/530 for more information.
+
+In order to make modifications to this repo, you'll have to [fork the repo](https://docs.github.com/en/get-started/quickstart/fork-a-repo) to your personal GitHub. There, you're free to make edits however you want. When actually creating a PR, however, try to match whole files at a time as PR reviewers are a scarce resource.
```

Comment:
No need to link the issue because it's just saying the same thing.
```suggestion
If you just want to get started and match a function, you don't need to create a PR! You can just [create an issue](https://github.com/doldecomp/melee/issues/new) with your decomp.me link and a maintainer will add your code to the repo.

If you're familiar with git and want to make changes locally, you can also [fork the repo](https://docs.github.com/en/get-started/quickstart/fork-a-repo) to your personal GitHub. When submitting code, try to group your changes into fewer but larger PRs, as it'a easier to review that way.
```